GVRf/Framework/backend_oculus/src/main/java/org/gearvrf/OvrViewManager.java

```diff
@@ -417,5 +417,6 @@ public void onRotationSensor(long timeStamp, float rotationW, float rotationX, f
         }
     }
 
-    private static native void drawEyes(long ptr);
+    private native void drawEyes(long ptr);
+    private native void readRenderResultNative(long renderTexture, Object readbackBuffer);
```

Comment:
No, intentional. Java's ``this`` is passed implicitly this way and it is needed by the native code.